app/workers/detect_invariants.rb

```diff
@@ -12,6 +12,7 @@ def perform
     detect_course_sync_not_succeeded_for_an_hour
     detect_high_sidekiq_retries_queue_length
     detect_application_choices_with_courses_from_the_incorrect_cycle
+    detect_submitted_applications_with_an_incorrect_number_of_selected_references
```

Comment:
Can be more specific here - `detect_submitted_applications_with_more_than_two_selected_references`